.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: required
+language: java
+jdk:
+  - oraclejdk8
+services:
+  - docker
+before_install:
+  - sudo apt-get -qq update
```

Comment:
`-qq` is quiet level 2, with some caveats that do not apply to an `apt-get update` (only `install` iirc). Define more readable?